vignettes/compareScenariosRemind2.Rmd

```diff
@@ -0,0 +1,199 @@
+---
+title: "compareScenarios in remind2"
+author: 
+ - "Christof Schötz"
+ - "Falk Benke"
+date: "2024-04-26"
+output: 
+  rmarkdown::html_vignette:
+    toc: true
+    number_sections: true
+vignette: >
+  %\VignetteIndexEntry{compareScenarios in remind2}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  eval = FALSE
+)
+```
+
+# Overview
+
+This tutorial explains how to visually compare the results of multiple runs of the IAM [REMIND](https://github.com/remindmodel/remind) using the framework [piamPlotComparison](https://github.com/pik-piam/piamPlotComparison).
+
+The common use case is creating plots after your REMIND runs have finished, using the  [`remind/output.R`](https://github.com/remindmodel/remind/blob/develop/output.R) script: Execute `Rscript output.R` in the REMIND folder, select `Comparison across runs` and then `compareScenarios2`.
```

Comment:
Will we keep `compareScenarios2` here or should we move to `compareScenarios` as well in remindmodel?

Reply:
I don't mind renaming it in the future. 
The ratioale behind keeping the name compareScenarios2 / cs2 for now is that it could be useful to differentiate between:
- `piamPlotComparison::compareScenarios`: a generic function independent of the reporting library (remind2, reporttransport, reportbrick, ...)
- `compareScenarios2` a function to call the compareScenarios framework in the context of the remind2 reporting library